### PR TITLE
New outsider alt title: Space Farer (outsider shuttles)

### PR DESCRIPTION
**Pull request by tacoguy7765093**
## About The Pull Request
Adds a new alt-title of outsider called Spacefarer, with a new special superposed pod, one that can contains a full overmap ship inside. Do note, that you need to be in space to deploy this ship but the outsiders come with jetpacks. Will make a unique spawn location for them in the future.

Currently only have 3 ships available for the public.

Pizza shuttle, fast, gas guzzler, contains kitchen, has fax machine for station communication
dreamseeker_88Ch3RLiVN

Cybershuttle, snobby, overrated, EXTREMELY FAST, a deathtrap, but.. looks expensive I guess.
dreamseeker_icGVQMCHt0

Whiteship mk2, with anomaly science and dorms.
dreamseeker_YW9mcdbyVt

## Changelog

:cl:
add: New outsider alt-title, space farer: Currently only 2 shuttles
maptweak: Added outsider shuttles, the pizzashuttle, and the cyber shuttle.
/:cl:

### CODE REVIEW

**Comment by TheCaramelion** [comment]
Please someone convince him to use the champagne for the shuttles :(

**Comment by tacoguy7765093** [comment]
It will break adminspawn shuttles if I have to create pods without the shuttle landmarkers, which can crash the game if inited, and its too cumbersome to players to need a _champagne bottle_ as part of their kit to be able to fly a shuttle.

File: modular_chomp/code/game/jobs/job/noncrew.dm [hunk 1]
```diff
@@ -1,29 +1,34 @@
...
+	account_allowed = 0
+	offmap_spawn = TRUE
+
+	outfit_type = /decl/hierarchy/outfit/noncrew
+	job_description = {"Players taking a role of an outsider not employed by NT with no special mechanics. One superpose pod is provided.
 		-----Server rules still apply to the fullest
 		-----Outsiders are considered unauthorized personnel on Southern Cross.
 		-----Outsiders are not allowed to take part in events and mini-event areas unless the EM says otherwise.
-		-----Outsiders are not allowed to take station jobs.
+		-----Outsiders are not allowed to take station jobs.gonna
```
Comment (Kashargul) [inline modular_chomp/code/game/jobs/job/noncrew.dm:22]: gonna?

File: modular_chomp/maps/overmap/om_ships/offmap.dm [hunk 2]
```diff
@@ -11,3 +11,5 @@
 #include "shelter_5.dm"
 #include "shelter_6.dm"
 #include "cruiser.dm"
+#include "pizzashuttle.dm"
+#include "cybershuttle.dm"
```
Comment (Kashargul) [inline modular_chomp/maps/overmap/om_ships/offmap.dm:15]: Isn't one ship missing here if it's 3?

**Pull request merged**